README.md

```diff
@@ -1,4 +1,35 @@
 # Monasca Helm
 
-Contains helm charts for our Monasca components as well as a top level Monasca helm chart that can be used to deploy
-Monasca.
\ No newline at end of file
+This repo contains Helm charts for Monasca and its dependencies. Each chart release is hosted on
+[monasca.io](http://monasca.io) via github pages.
+
+## Quick Start
+
+To install Monasca in Kubernetes you can follow the following steps:
+
+```bash
+$ helm repo add monasca http://monasca.io/monasca-helm-repo
+$ helm install monasca/monasca --name monasca --namespace monitoring
+```
+
+By default Monasca will monitor pod workloads, basic Kubernetes Health and autodetect Prometheus endpoints.
```

Comment:
Another thought, maybe it would be good to say something like "it may take several minutes for all containers to download" and that it's normal to see a pod restart or two while things are initializing, especially on slower connections.